src/Form/Field.php

```diff
@@ -304,6 +323,16 @@ public function formFields(): Fields
 		return $this->siblings;
 	}
 
+	protected function handlerCall(string $handler, ...$args): mixed
+	{
+		return $this->options[$handler]->call($this, ...$args);
+	}
+
+	protected function handlerExists(string $handler): bool
+	{
+		return isset($this->options[$handler]) === true && $this->options[$handler] instanceof Closure;
+	}
```

Comment:
`Feedback:` Personally, to me this optimization makes the code harder to read and understand while not keeping it that much more DRY. For me, it would be easier to read if we leave the actual checks and calls in the methods, but maybe it's different for you and/or others.